settings.py

```diff
@@ -19,6 +19,7 @@
     'default': dj_database_url.config(),
 }
 DATABASES['other'] = copy.deepcopy(DATABASES['default'])
-DATABASES['other']['NAME'] += '_other'
+if 'NAME' in DATABASES['other']:
+    DATABASES['other']['NAME'] += '_other'
```

Comment:
One could argue that this is kinda silly - why even bother with the "other" DB if there's not even any configuration for the first DB in the first place :man_shrugging:

Reply:
I'm fine with it. This is an unfortunate hack of running tests with multiple databases and satisfying circle and all of these other things